setup.py

```diff
@@ -29,7 +29,6 @@
         "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
         "Operating System :: OS Independent",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2.6",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
```

Comment:
any reason to not add 3.7/3.8 here?

Reply:
Those are already there a few lines down. :-)

Reply:
oops :D